src/core/main.c

```diff
@@ -79,6 +79,7 @@
 #include "util.h"
 #include "virt.h"
 #include "watchdog.h"
+#include <setjmp.h>
```

Comment:
Please put this alphabetically among the system headers at the top (those with `<>`).